lib/Widget/evaluationForms/post-hos/post-hos-day2-7/pulmanary_form.dart

```diff
@@ -391,19 +389,11 @@ class _PulmanaryFormState extends State<PulmanaryForm> {
                                                 result = "Pass";
                                               } else {
                                                 result = "NotPass";
-
-                                                // var creation = _calculationService.formatDate(
-                                                //     date: DateTime.now());
-                                                // var patientState = _anSubCollection['state'];
-                                                // _firebaseService.addNotification({
-                                                //   'formName': 'Pulmanary',
-                                                //   'formId': formId,
-                                                //   'userId':
-                                                //       UserStore.getValueFromStore('storedUserId'),
-                                                //   'creation': creation,
-                                                //   'patientState': patientState,
-                                                //   'seen': false,
-                                                // });
+                                                _firebaseService
+                                                    .addNotification(
+                                                        hn: widget.hn,
+                                                        formId: formId,
+                                                        formName: 'Pulmanary');
```

Comment:
Do we need to store Failed form to Form Collection ?

Reply:
We need the data plot the graph and still need to keep in the notification collection to show in notification page of medical team side

Reply:
so FirebaseService.addNotifaction() doesn't help in stroing the data to `FormsCollection`. You might need to call some funct to store it.

Reply:
<img width="268" alt="Screen Shot 2564-04-11 at 20 14 38" src="https://user-images.githubusercontent.com/57941667/114305509-8ba4b800-9b02-11eb-81e6-40abff06e815.png">

It has already